systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -202,10 +202,10 @@ public static JsonArray expectedServiceDiscoveryInfo(int port, String protocol,
         return jsonArray;
     }
 
-    public static JsonArray expectedServiceDiscoveryInfo(String plainAuth, String tlsAuth) {
+    public static JsonArray expectedServiceDiscoveryInfo(String plainAuth, String tlsAuth, boolean plainTls, boolean tlsTls) {
```

Comment:
This looks little bit weird for me -> if you have `plainAuth`, you can simply just add the `true` and you don't have to add another boolean ... same for the `tls`.. and if you want to keep it like this the names are bit confusing `plainTls` and `tlsTls`..

Reply:
it makes sense for plain, but how do you want to determine form one value, that you have tls encryption on the listener without authentication?

Reply:
hmm okay, that makes sense, so maybe add some javadoc explaining the `plainTls` and `tlsTls` -> that it's for auth and encryption?

Reply:
sorry, I didn't figured it out just by the names :D but I guess that's problem on my side

Reply:
Ye, make sense.